src/sync/incomingSyncUtils.js

```diff
@@ -608,18 +608,26 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
       break;
     }
     case 'Transaction': {
-      if (record.store_ID !== settings.get(THIS_STORE_ID)) break; // Not for this store
+      // Only sync transactions for this store.
+      const { store_ID } = record;
+      if (store_ID !== settings.get(THIS_STORE_ID)) break;
```

Comment:
In cases like this I don't really get the improvement. What was readable on 1 line is now 2 lines. I'd understand if store_ID was used again or if you were looking to use other properties of `record`.

To follow this through properly shouldn't you grab every other property used in this case statement (i.e. all properties of `record`)?

Reply:
Yeah, I think its good practice in general to destructure instead of dot notation. It's consistent throughout the code base and it also has the nice self-documenting property of showign at the top of the method what properties it uses. Will add the other destructured variables.

Reply:
I think we should refrain from refactoring code full stop